.storybook/styles/chromatic.css

```diff
@@ -5,5 +5,5 @@
 *,
 *::before,
 *::after {
-    font-family: Arial;
+    font-family: Arial !important;
```

Comment:
why !important? :( is there any other way?

Reply:
It's only used to make sure that in Chromatic tests we always use Arial as loading a custom font generates issues.

The important is there to ensure no specificity wars with the basecode.